### PR TITLE
Added sorting to quizzes tab

### PR DESCRIPTION
**Pull request by TSlashDreamy**
Preview:
зображення

Lint:
зображення

### CODE REVIEW

**Review by Yurenko — APPROVED** [review]

File: src/containers/my-quizzes/QuizzesContainer.tsx [hunk 1]
```diff
@@ -63,25 +69,18 @@ const TestsContainer = () => {
...
-        />
-      )}
-    </Box>
+  const tableAttachments = (
+    <EnhancedTable
+      columns={columnsToShow}
+      data={{ items: response.items }}
+      emptyTableKey='myResourcesPage.quizzes.emptyQuizzes'
+      rowActions={rowActions}
+      sort={{ sort, onRequestSort }}
```
Comment (Tolik170) [inline src/containers/my-quizzes/QuizzesContainer.tsx:78]: you can pass sortOptions here: 
sort={sortOptions}

**Review by ArturBekhDEV — APPROVED** [review]

File: src/containers/my-quizzes/QuizzesContainer.tsx (hunk 1 above)
Comment (TSlashDreamy) [reply src/containers/my-quizzes/QuizzesContainer.tsx]: > you can pass sortOptions here: sort={sortOptions}

fixed that and removed unnecessary variable

**Review by Yurenko — APPROVED** [review]

File: src/containers/my-quizzes/QuizzesContainer.tsx [hunk 2]
```diff
@@ -108,25 +114,18 @@ const TestsContainer = () => {
...
-          rowActions={rowActions}
-          sort={{
-            sort: { order: SortEnum.Desc, orderBy: 'updatedAt' },
-            onRequestSort: () => null
-          }}
-          sx={styles.table}
-        />
-      )}
-    </Box>
+  const tableAttachments = (
```
Comment (ArturBekhDEV) [inline src/containers/my-quizzes/QuizzesContainer.tsx:117]: I think it's not table of attachments :)

Comment (TSlashDreamy) [reply src/containers/my-quizzes/QuizzesContainer.tsx]: > I think it's not table of attachments :)

You are absolutely right, i missed that, thank you, fixed.

File: src/containers/my-quizzes/QuizzesContainer.tsx [hunk 3]
```diff
@@ -8,23 +8,28 @@ import { useSnackBarContext } from '~/context/snackbar-context'
 import EnhancedTable from '~/components/enhanced-table/EnhancedTable'
 import Loader from '~/components/loader/Loader'
 import AddResourceWithInput from '~/containers/my-resources/add-resource-with-input/AddResourceWithInput'
+import useSort from '~/hooks/table/use-sort'
```
Comment (mxrcury) [inline src/containers/my-quizzes/QuizzesContainer.tsx:11]: fix this warning

Comment (TSlashDreamy) [reply src/containers/my-quizzes/QuizzesContainer.tsx:10]: > fix this warning

fixed

**Pull request merged**